sounds.lua

```diff
@@ -0,0 +1,48 @@
+--[[
```

Comment:
This file is pretty copy-paste heavy. I don't see why these need to be functions, and why this can't be just a `sounds` table: `return {dirt = ..., wood = ...}`. If you want to get `default` defaults then, you could simply loop:

```lua
local sounds = {}
if has_default_mod then
  for _, sound in ipairs{"dirt", "wood", "stone", "glass", "leaves"} do
    sounds[sound] = default["node_sound_" .. sound .. "_defaults"]()
  end
end
return sounds
```

Reply:
Fair point, i wasn't sure if that was too much repetition :+1: 
I applied your suggestion partially: the sound-function remains as it is used everywhere like this,
the functions now get applied in a loop over the various sound-names, still readable but the code is much smaller
and could be extended if more sound-names would be added (it won't, but still...)